specification/cognitiveservices/data-plane/ContentModerator/v1.0/contentmoderatorListManagement.swagger.json

```diff
@@ -0,0 +1,1446 @@
+        "description": "Deletes all terms from the list with list Id equal to the list Id passed.",
+        "parameters": [
+          {
+            "name": "listId",
+            "in": "path",
+            "description": "Id of the term list.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "language",
+            "in": "query",
+            "description": "Language of the terms.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ocpApimSubscriptionKeyParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "string"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete AllTerms request": {
+            "$ref": "./examples/DeleteAllTermsResource.json"
+          }
+        },
+        "produces": [
+          "application/json"
+        ]
+      }
+    }
+  },
+  "definitions": {
+    "CreateImageListResponse": {
+      "description": "Create Image List Response.",
+      "properties": {
+        "id": {
+          "description": "Image List Id.",
+          "type": "number"
+        },
+        "name": {
+          "description": "Image List Name.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Description for image list.",
+          "type": "string"
+        },
+        "metadata": {
+          "description": "Image List Metadata.",
+          "type": "object",
+
+          "properties": {
+            "sample string 1": {
+              "description": "Optional Key value pair to describe your list.",
+              "type": "string"
+            },
+            "sample string 2": {
+              "description": "Optional Key value pair to describe your list.",
+              "type": "string"
+            }
+          }
+
+        }
+      }
+    },
+    "CreateTermListResponse": {
+      "description": "Create Term List Response.",
+      "properties": {
+        "id": {
+          "description": "Term list Id.",
+          "type": "number"
+        },
+        "name": {
+          "description": "Term list name.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Description for term list.",
+          "type": "string"
+        },
+        "metadata": {
+          "description": "Term list metadata.",
+          "type": "object",
+          "properties": {
+            "sample string 1": {
+              "description": "Optional Key value pair to describe your list.",
+              "type": "string"
+            },
+            "sample string 2": {
+              "description": "Optional Key value pair to describe your list.",
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "ImageListGetDetailsResponse": {
```

Comment:
the name of this ends up being a class name -- I think this could be just `ImageListDetails` -- having the operation name and the word `Response` is a bit redundant.